### PR TITLE
Pulling dependencies up to dependencyManagement on parent pom

### PR DESCRIPTION
**Pull request by mbarrientos**
Pulling up dependencies and also removing some duplicated declarations

### CODE REVIEW

**Comment by adriannieto** [comment]
@andreaturli could you review this?

File: pom.xml [hunk 1]
```diff
@@ -92,17 +92,50 @@
...
+        <commons-httpclient.version>3.1</commons-httpclient.version>
+        <toscamodel.version>1.0.0</toscamodel.version>
+        <hsqldb.version>2.3.0</hsqldb.version>
+        <mysql-connector.version>5.1.27</mysql-connector.version>
+        <modaclouds.version>2.4</modaclouds.version>
+        <jaxb.version>2.2.6</jaxb.version>
+        <castor.version>1.2</castor.version>
+        
+        <!-- Test dependencies -->
         <junit.version>4.11</junit.version>
```
Comment (andreaturli) [inline pom.xml]: I think we should take the change to choose between junit or testng as having both already created some headaches.
I vote for testng as it supports groups more easily than junit, but not strong feelings.

No need to address this comment in this PR

File: pom.xml [hunk 2]
```diff
@@ -331,27 +337,96 @@
                     <goals>deploy</goals>
                 </configuration>
             </plugin>
-            <plugin>
```
Comment (andreaturli) [inline pom.xml]: why do you want to remove this plugin?

File: sla/sla-core/pom.xml [hunk 3]
```diff
@@ -144,13 +127,13 @@
 		<dependency>
 			<groupId>org.slf4j</groupId>
 			<artifactId>slf4j-simple</artifactId>
-			<version>${slf4j-version}</version>
```
Comment (andreaturli) [inline sla/sla-core/pom.xml]: good catch

File: pom.xml (hunk 2 above)
Comment (mbarrientos) [inline pom.xml]: It was duplicated right above it (https://github.com/SeaCloudsEU/SeaCloudsPlatform/pull/83/files#diff-600376dffeb79835ede4a0b285078036R318)

Comment (andreaturli) [inline pom.xml]: right! missed that, thanks!

**Comment by andreaturli** [comment]
+1

thanks @mbarrientos!

**Pull request merged**